### DISCUSSION TITLE
Basic build of FB 3490 V 07.1x - change Password fails

### DISCUSSION THREAD

**Discussion by JohnPaden** [Freetz-NG/freetz-ng#149] [Allgemein] [answered]
VM: Freetz-Linux-1.5.8-Ubuntu 20.04.01 LTS 
Basic build of FB 3490 V 07.1x via git clone https://github.com/Freetz-NG/freetz-ng ~/freetz-ng

Minimast Changes: Level of Competence: expert, Hardware: 3490, Firmware 07.1x

Logon to AVM FB works
Switch to 'Access via local LAN'
Access to Freetz works with default IDs
Change of PW results in :81 no longer accessable until reboot
>'Starting WebCFG failed'

Any hint? (And is this now the right place for such discussions?)

change: Same issue on 7580.

**Comment by fda77** [comment, marked as answer]
Should be fixed with https://github.com/Freetz-NG/freetz-ng/commit/f2f5e8430e88db4fed98a43790403adc9ba9d2e0
See also https://github.com/Freetz-NG/freetz-ng/issues/153

**Reply by JohnPaden** [reply]
Klasse - danke!!!
